pcsx2/DEV9/InternalServers/DHCP_Server.cpp

```diff
@@ -291,7 +291,7 @@ namespace InternalServers
 			//otherwise use adapter's DNS1
 
 			if (dnsIPs.size() >= 1)
-				dns2 = dnsIPs[std::min((size_t)2, dnsIPs.size())];
+				dns2 = dnsIPs[std::min((size_t)2, dnsIPs.size()) - 1];
```

Comment:
Might be a little more readable like this:
(multi line suggestion doesn't work on unchanged lines, if-statement applies to line above)
```suggestion
			if (!dnsIPs.empty())
				dns2 = dnsIPs[std::min<size_t>(1, dnsIPs.size() - 1)];
```